src/interface/src/app/plan/scenario-details/outcome/outcome.component.ts

```diff
@@ -35,7 +41,17 @@ export class OutcomeComponent implements OnInit, OnChanges {
   }
 
   ngOnInit(): void {
-    this.authService.loggedInUser$.pipe(take(1)).subscribe(this.currentUser$)
+    this.authService.loggedInUser$.pipe(take(1)).subscribe((user) => {
+      if (user) {
+        this.currentUser$.next(user);
+        this.displayName = user.firstName ? user.firstName : (user.username ? user.username : '');
```

Comment:
nit: I think this can be shortened to

`user.firstName ? user.firstName : (user.username ?? '')`

Reply:
thanks! done